build/scripts/check-toolset-insertion.ps1

```diff
@@ -21,6 +21,11 @@ try
         "Microsoft.Build.Utilities.Core.dll",
         "Microsoft.DiaSymReader.Native.arm.dll"
     )
+
+    $exludedFromVsix = @(    
+        "Microsoft.DiaSymReader.Native.x86.dll", # installed by msbuild setup component
+        "Microsoft.DiaSymReader.Native.amd64.dll"  # installed by msbuild setup component
```

Comment:
Ah, something is wrong. The compiler package shouldn't change. Only the IDE one. Looking...